packages/authorization-process/package.json

```diff
@@ -42,8 +42,9 @@
     "openapi-zod-client": "1.18.1",
     "pagopa-interop-commons": "workspace:*",
     "pagopa-interop-models": "workspace:*",
-    "pg-promise": "11.8.0",
-    "ts-pattern": "5.2.0",
+    "pagopa-interop-selfcare-v2-client": "workspace:*",
+    "pg-promise": "11.6.0",
+    "ts-pattern": "5.1.1",
```

Comment:
Is there a reason for this change? I would keep the same versions we use in the other services